recipes/libassert/all/conanfile.py

```diff
@@ -109,7 +118,7 @@ def package(self):
             copy(
                 self,
                 "*.dll",
-                src=self.build_folder,
+                src=os.path.join(self.package_folder, "lib"),
```

Comment:
Why needed to change it now? I see the version 2.0.0 install in the bin folder already.

Reply:
I don't quite recall, I'll try reverting it and see if anything breaks